commit 58842a113a66ab10c1ed1c1d9d0c5158a605995f (refs/changes/06/3506/3)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-02-27T15:44:50-08:00 (1 year, 7 months ago)
    
    TRITON-189 change in TRITON-184 broke config-agent running in zones
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
    Approved by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/agent.js b/agent.js
index 9f94266..f07a1fe 100644
--- a/agent.js
+++ b/agent.js
@@ -135,7 +135,8 @@ async.waterfall([
 				// Allow the config file to specify the
 				// instance UUID(s). This is used for load
 				// testing.
-				if (config.instances.length === 0) {
+				if (!config.instances ||
+				    config.instances.length === 0) {
 					config.instances = [ zonename ];
 				}
 			}
